python/pyproject.toml

```
@@ -0,0 +1,27 @@
+[build-system]
+requires = ["setuptools>=42", "cython", "cmake"]
+build-backend = "setuptools.build_meta"
+
+[tool.cibuildwheel]
+build = 'cp3{9,10}-manylinux_x86_64'
+manylinux-x86_64-image = 'manylinux_2_28'
+before-all = [
```

```suggestion
linux.before-all = [
```

(You can also put these in `[tool.cibuildwheel.linux]` if you like that better, vs. adding `linux.` to the front)